### PR TITLE
bz1785021 - using ssh to reach bootstrap node

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1785021

@duanwei33, PTAL.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@duanwei33, PTAL.

**Comment by kalexand-rh** [comment]
@duanwei33, PTAL?

File: modules/ssh-agent-using.adoc [hunk 1]
```diff
@@ -76,7 +76,7 @@ endif::[]
 [id="ssh-agent-using_{context}"]
 = Generating an SSH private key and adding it to the agent
 
-If you want to perform installation debugging or disaster recovery on your cluster, provide an SSH key to your `ssh-agent` and installation program.
+If you want to perform installation debugging or disaster recovery on your cluster, you must provide an SSH key to both your `ssh-agent` and to the installation program. You can use this key to access the bootstrap machine in a public cluster to troubleshoot installation issues.
```
Comment (duanwei33) [inline modules/ssh-agent-using.adoc:79]: > to both your `ssh-agent` and to the installation program.
The second "to" is not necessary here?

**Comment by duanwei33** [comment]
@kalexand-rh Really sorry that I missed the notification. seems ok to add a note to connect bootstrap node. (Just tiny comments to confirm :) ) 

@gpei Could you help have a check as well?

**Comment by gpei** [comment]
LGTM

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**